skyportal/handlers/api/observation.py

```diff
@@ -257,44 +269,14 @@ def get(self):
                   schema: Error
         """
 
-        data = self.get_json()
-
-        telescope_name = data.get('telescope_name')
-        instrument_name = data.get('instrument_name')
-        start_date = data.get('start_date')
-        end_date = data.get('end_date')
-        localization_dateobs = data.get('localizationDateobs', None)
-        localization_name = data.get('localizationName', None)
-        localization_cumprob = data.get("localizationCumprob", 0.95)
-        return_probability = data.get("returnProbability", False)
-
-        telescope = (
-            Telescope.query_records_accessible_by(
-                self.current_user,
-            )
-            .filter(
-                Telescope.name == telescope_name,
-            )
-            .first()
-        )
-        if telescope is None:
-            return self.error(message=f"Missing telescope {telescope_name}")
-
-        instrument = (
-            Instrument.query_records_accessible_by(
-                self.current_user,
-                options=[
-                    joinedload(Instrument.fields),
-                ],
-            )
-            .filter(
-                Instrument.telescope == telescope,
-                Instrument.name == instrument_name,
-            )
-            .first()
-        )
-        if instrument is None:
-            return self.error(message=f"Missing instrument {instrument_name}")
+        telescope_name = self.get_query_argument('telescope_name', None)
+        instrument_name = self.get_query_argument('instrument_name', None)
```

Comment:
why not make these inputs camel case also?

Reply:
Done!